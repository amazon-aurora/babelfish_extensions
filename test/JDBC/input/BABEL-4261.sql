CREATE TABLE t_babel4261 (a int, b int);
GO

select set_config('parallel_setup_cost', 0, false);
select set_config('parallel_tuple_cost', 0, false);
select set_config('min_parallel_table_scan_size', 0, false);
GO

select a, count(*) from t_babel4261 group by a order by 2; -- should not crash
GO

DROP TABLE t_babel4261;
GO

