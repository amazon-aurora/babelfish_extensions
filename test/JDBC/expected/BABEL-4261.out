CREATE TABLE t_babel4261 (a int, b int);
GO

select set_config('parallel_setup_cost', 0, false);
select set_config('parallel_tuple_cost', 0, false);
select set_config('min_parallel_table_scan_size', 0, false);
GO
~~START~~
text
0
~~END~~

~~START~~
text
0
~~END~~

~~START~~
text
0
~~END~~


-- execution
select a, count(*) from t_babel4261 group by a order by 2; -- should not crash
GO
~~START~~
int#!#int
~~END~~


DROP TABLE t_babel4261;
GO

